apps/cookbook/src/app/showcase/header-showcase/header-showcase.component.html

```diff
@@ -0,0 +1,67 @@
+<div class="example">
+  <div class="header-example-container">
+    <p>
+      Header is a component that manages the layout of the header section of kirby-page. It is
+      responsive and will change the layout according to the device size. It allows for a wide range
+      of use cases including support for avatar, flags etc.
+    </p>
+
+    <h2>Default</h2>
+
+    <cookbook-example-viewer [html]="defaultExample.template">
+      <div class="header-example">
+        <cookbook-header-example-default #defaultExample></cookbook-header-example-default>
+      </div>
+    </cookbook-example-viewer>
+  </div>
+
+  <div class="header-example-container">
+    <h2>Flag</h2>
+    <p>
+      A flag can be used to indicate a status. To use flag you simply add
+      <code>kirby-flag</code>
+      in the header and that should position the flag correctly and manage spacing.
+    </p>
+    <cookbook-example-viewer [html]="flagExample.template">
+      <div class="header-example">
+        <cookbook-header-example-flag #flagExample></cookbook-header-example-flag>
+      </div>
+    </cookbook-example-viewer>
+  </div>
+
+  <div class="header-example-container">
+    <h2>Key value</h2>
+    <p>
+      In some cases a number is the primary information on a page. When that's the case you should
+      use the `value` prop
```

Comment:
Consider formatting:
```suggestion
      use the <code>value</code> prop
```